autogpt/configurator.py

```diff
@@ -14,9 +14,6 @@
 if TYPE_CHECKING:
     from autogpt.config import Config
 
-GPT_4_MODEL = "gpt-4"
-GPT_3_MODEL = "gpt-3.5-turbo"
```

Comment:
Honestly I'd like to keep these constants but move them to `config.py` and rename to `DEFAULT_SMART_LLM` and `DEFAULT_FAST_LLM`